frontend/src/utils/helpers/isCourseCompleted.ts

```diff
@@ -0,0 +1,17 @@
+import { Course } from 'types/course';
+
+const isCourseCompleted = (course: Course): boolean => {
+  let completed = true;
+  for (let material = 0; material < course.materials.length; material++) {
+    const content = course.materials[material].content;
+    for (let stage = 0; stage < content.length; stage++) {
+      if (!content[stage].isCompleted) {
+        completed = false;
+        break;
+      }
+    }
+  }
+  return completed;
```

Comment:
Maybe it will be better to define at first `completed = false` and check the last material content stage?